source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/readonly/ImageRenderer.java

```diff
@@ -221,6 +221,7 @@ private static ConstraintSet createConstraints(Context context, ImageView imageV
         // Start with base constraints from XML layout
         constraints.clone(context, R.layout.image_constraint_layout);
 
+        // TODO: Check RTL support
```

Comment:
you may want to create an issue and include the # in this comment to track.

Reply:
Fixed